source/administration/changelog.md

```diff
@@ -170,6 +294,7 @@ Mattermost v5.31.0 contains a low level security fix. [Upgrading](https://docs.m
  
 ### Known Issues
  - The Database Schema Version is displayed as 5.30.0 in the About Mattermost modal.
+ - Config.json can reset when running any ``sudo -u mattermost ./bin/mattermost`` commands or when running ``systemctl restart mattermost`` [MM-32390](https://mattermost.atlassian.net/browse/MM-32390).
```

Comment:
This is not necessarily true. Only affected commands would be those that write to config (e.g. `config` or `plugin`). For example, we [fixed](https://github.com/mattermost/mattermost-server/pull/16805) other frequently used commands like `version` to be read-only so they won't cause this issue. Also, I am curious about the `systemctl restart mattermost` case. This wasn't reported originally and I am not sure how it could be causing the issue.

Reply:
Thanks, I've fixed the text.

Reply:
@streamer45  I added this ticket to known issues: https://mattermost.atlassian.net/browse/MM-33752